src/actions/profilesActions.ts

```diff
@@ -120,3 +126,87 @@ export function verifySocialAccount(accountAddress: string, account: IProfileSta
     });
   };
 }
+
+
+export function serverLoginByEthSign(accountAddress: string) {
```

Comment:
Remove the duplicate code in AccountProfilePage.tsx?

Reply:
yes done